src/main/java/edu/harvard/iq/dataverse/MailServiceBean.java

```diff
@@ -221,12 +217,30 @@ public void sendMail(String reply, String to, String subject, String messageText
             me.printStackTrace(System.out);
         }
     }
-    
-    public Boolean sendNotificationEmail(UserNotification notification){  
+
+    /**
+     * Set the contact delegation as "[dataverse team] on behalf of [user email]"
+     * @param reply The user's email address as give via the contact form
+     * @param fromAddress The system email address
+     * @throws UnsupportedEncodingException
+     */
+    public void setContactDelegation(String reply, InternetAddress fromAddress)
+            throws UnsupportedEncodingException {
+        String personal = fromAddress.getPersonal();
+        if (personal == null)
+            personal = "Dataverse administrator";
```

Comment:
Not a full review, just yet, but can we put this in the bundle? (also let's call it as "Dataverse Admin")

Reply:
Actually, maybe a better alternative, rather than in a bundle, what if it used the branding name of the installation (using ```BrandingUtil.getInstallationBrandName()```)? Thoughts @qqmyers @djbrooke

Reply:
Dataverse Installation Administrator is better than "null" so I'd be happy to accept a PR with that change, but if @pkiraly would be willing to add it, the branding name would be a better option.

Reply:
That makes sense. There are also a getSupportTeamName()/getSupportTeamEmail() methods that might help -the former uses the brand name.

Reply:
I made another version which solves it in a three step process:
```
        String personal = fromAddress.getPersonal() != null
            ? fromAddress.getPersonal()
            : BrandingUtil.getInstallationBrandName() != null
                ? BrandingUtil.getInstallationBrandName()
                : BundleUtil.getStringFromBundle("contact.delegation.default_personal");
```
In order to test it with unit tests I had to add some extra null check into getInstallationBrandName() method.

Reply:
I have a question. It seems that tests are not independent from each other. If I run only the tests written for this feature (`mvn clean test -Dtest=MailServiceBeanTest`), it goes well (BrandingUtil.getInstallationBrandName() returns null), however when it is run among other tests, BrandingUtil.getInstallationBrandName() returns "LibraScholar". It seems some other tests inject values, and then missed to set it back into a blank state.
Do you have an idea how to resolve this problem?

Reply:
Sorry - I think there are two options. BrandingUtil requires some setup to work in tests (see the BrandingUtilTest class). 

1. Add a line like  https://github.com/IQSS/dataverse/blob/473cde69816714a7db42a90343932abec4f118b9/src/test/java/edu/harvard/iq/dataverse/branding/BrandingUtilTest.java#L57 after line 62 resetting the return value to null, and probably to the MailUtilTest as well, and/or
2. Add https://github.com/IQSS/dataverse/blob/473cde69816714a7db42a90343932abec4f118b9/src/test/java/edu/harvard/iq/dataverse/branding/BrandingUtilTest.java#L35-L39 to your test to configure BrandingUtil with the values you want.

(Nominally, "LibraData" was the value that was returned prior to the recent updates to Dataverse to have BrandingUtil use services. With only BrandingUtilTest and MailUtilTest using BrandingUtil, I didn't worry about resetting for other tests. I guess that (1) would be cleaner, but doing (2) in your own tests should always work as well.)

Reply:
@qqmyers I tried both. (1) did not work at all, but (2) did with a modification. Mockito.when throws an UnnecessaryStubbingException. After some searching I found https://www.baeldung.com/mockito-unnecessary-stubbing-exception, which suggested to use `Mockito.lenient().when` and it worked. Now tests covers 4 cases:
* the :SystemEmail contains a name part
* if not, there is an :InstallationName
* if not, the root Dataverse has a name
* if not, there is a bundle property called `contact.delegation.default_personal`

Reply:
@pkiraly - glad you got it to work. Thinking back, that stubbing exception may have been why I didn't do (1) myself. I can see why in (2), where you have the setup in a private method you could get the same exception - seems like Mockito is just checking whether the mock is used later in the same method - so I'm glad lenient works there. (Did you try lenient in (1)? Just curious - not suggesting that you need to modify the PR).

Reply:
@qqmyers No, with (1) there were no effect at all, my impression was, that something did not happened at all. One more point: I wanted that these tests should work both alone and together with all the other tests. I guess when it was running alone it did not triggered BrandingUtilTest tests.

Reply:
Hmm - I just tried (1) and get the unnecessary stubbing error - when the BrandingUtilTest runs. Using lenient() there gets rid of the warning, but I didn't test whether that would have any effect on other test classes (perhaps lenient() just eats the warning and doesn't actually reset the mock at all - which would be consistent if you see no effect of (1)).

In any case, I think what you have now works when testing one class or all of them and that's what needed.